src/banker.rs

```diff
@@ -49,8 +49,6 @@ pub async fn message(msg: String) -> Result<(), String> {
 }
 
 pub async fn invoice(user: &str, amount: u64, reason: &str) -> Result<(), String> {
-    println!("Invoicing user {} for {} gp", user, amount);
```

Comment:
That wasn't to say that this shouldn't be logged - just not as a `println!` macro

Reply:
I thought I'd remove it anyway; I'll be rewriting this module in a bit anyway